src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxesSelectFieldsFromTable.class.php

```diff
@@ -14,6 +14,27 @@
 /**/
 class TCMSFieldLookupMultiselectCheckboxesSelectFieldsFromTable extends TCMSFieldLookupMultiselectCheckboxes
 {
+    protected function GetMLTFilterQuery()
+    {
+        $filterQuery = parent::GetMLTFilterQuery();
+
+        // NOTE these conditions are _very_ specific to the current configuration of cms_field_conf
+
+        if (false !== \strpos($filterQuery, 'AS tabname') && false !== \strpos($filterQuery, 'cms_field_conf')) {
+            $orderClauseCheck = 'ORDER BY position ASC';
+            $orderPosition = \strpos($filterQuery, $orderClauseCheck);
+
+            // TODO "tabname" is the configuration "group field" or "group field column name" of the table - however that must be present in the custom list query.
```

Comment:
Looks like this is always the case. Because it is THE most core like table... there should not be changes made here by users i think. As long as we check for the existence of tabname this is not breaking things here.

Reply:
I meant this more like there is a possibility to do this for other cases dynamically (correct).

I removed the comment.